lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/dcmawCri/ContractTest.java

```diff
@@ -1219,92 +1219,6 @@ void fetchVerifiableCredential_whenCalledAgainstDcmawCri_retrievesAValidNldPassp
                         });
     }
 
-    @Test
```

Comment:
This has just moved down to below the pact it uses. The test hasn't actually changed